src/routes/Tool/loader.ts

```diff
@@ -4,16 +4,20 @@ import { removeDuplicates } from '@/utils';
 import { LoaderFunction } from 'react-router-dom';
 
 export interface ToolLoaderData {
-  tyomahdollisuudet: components['schemas']['EhdotusDto'][];
+  tyomahdollisuudet: components['schemas']['TyomahdollisuusDto'][];
   osaamiset: components['schemas']['YksilonOsaaminenDto'][];
 }
 
 export default (async ({ request }) => {
-  const { data: tyomahdollisuudet } = await client.POST('/api/ehdotus/tyomahdollisuudet', {
-    body: {},
+  let tyomahdollisuudet: components['schemas']['TyomahdollisuusDto'][] = [];
+
+  const { data } = await client.GET('/api/tyomahdollisuudet', {
+    params: {},
```

Comment:
Should take into account the competences of the logged in user. See current main version.